dask/cli.py

```diff
@@ -40,6 +40,36 @@ def versions():
     show_versions()
 
 
+@cli.group()
+def config():
+    """Dask config settings"""
+    pass
+
+
+@config.command()
+@click.argument("key", default=None, required=False)
+def get(key=None):
+    """Print config key, or the whole config."""
+    from functools import reduce
+
+    from yaml import dump
+
+    from dask.config import config
+
+    if key is None:
+        click.echo_via_pager(dump(config))
+    else:
+        try:
+            data = reduce(lambda d, k: d[k], key.split("."), config)
+            if isinstance(data, (list, dict)):
+                click.echo_via_pager(dump(data))
+            else:
+                click.echo(data)
+        except KeyError:
+            click.echo(click.style(f"Section not found: {key}", fg="red"), err=True)
```

Comment:
I was thinking about whether it would be better to return nothing as suggested in #9746 (and e.g. as `git config --get` does), but I think I like that we highlight the difference between a key set to `null` (or `""`) and the absence of a key.

Reply:
I know, I could do the same as `git config` does, and add two commands, one for "get" and one for "list". If a "get" is called without key, it would error out (print usage and exit). I opted for this one, because I think it's more useful than throwing an error. But it's a matter of preference. I always find myself struggling to remember how to see the whole config: is it `get`, `list`, `show`? Less commands seems better.

Reply:
Eh, let's do `get` and `list`. It's growing on me.